### PR TITLE
Oscar Shuttle - a Custodial Ship

### PR DESCRIPTION
**Pull request by kennedytcooper**
## About the PR

Clean up the galaxy with the Oscar, a ship for trashy people by which I mean Janitors! Extraordinarily well-stocked but a bit cramped not necessarily the _safest_ working conditions ever created, but it's cheap!

**Media**

Screenshot from 2023-08-04 11-41-24

**Changelog**

:cl:
- add: the Oscar, a small ship for Janitors

### CODE REVIEW

**Comment by sTiKyt** [comment]
A bit weird they are all the same but with different name cuz of interior?

Would make more sense if they had some shared "Model" name and Interrior's name suffix

Like "Whatever - Custodial MK"

**Comment by kennedytcooper** [comment]
Oscar is actually a completely unique design, only Bocadillo and Quimicita are the same and that's specifically for meth RV memes

**Review by Cheackraze — APPROVED** [review]

File: Resources/Prototypes/_NF/Shipyard/oscar.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+- type: vessel
+  id: Oscar
+  name: NC Oscar
+  description: A little shuttle for
```
Comment (Cheackraze) [inline Resources/Prototypes/_NF/Shipyard/oscar.yml:4]: Description seems to have been cut off

**Comment by Cheackraze** [comment]
feel free to reopen if you come back

**Pull request closed without merging**